webpack.config.js

```diff
@@ -91,4 +92,48 @@ const config = {
         ],
     },
 }
-module.exports = config
+
+/**@type {import('webpack').Configuration}*/
+const webviewConfig = {
```

Comment:
Note: may look to move this logic to a different job (or invocation). This is required prior to testing the extension, and cannot be triggered right now without a full webpack.

Reply:
> Note: may look to move this logic to a different job (or invocation). 

"job" meaning `npm run` task or CI job or...?

> This is required prior to testing the extension, and cannot be triggered right now without a full webpack.

Example of a non-full webpack? What does "testing" mean here exactly?